include/swift/IDE/CodeCompletionResult.h

```diff
@@ -545,9 +548,11 @@ class ContextFreeCodeCompletionResult {
 
   const CodeCompletionResultType &getResultType() const { return ResultType; }
 
-  ContextFreeNotRecommendedReason getNotRecommendedReason() const {
-    return NotRecommended;
-  }
+  ContextFreeNotRecommendedReason getNotRecommendedReason() const { return NotRecommended; }
+
+  ContextualNotRecommendedReason calculateContextualNotRecommendedReason(ContextualNotRecommendedReason explicitReason, bool canCurrDeclContextHandleAsync) const;
+
+  CodeCompletionResultTypeRelation calculateContextualTypeRelation(const DeclContext *dc, const ExpectedTypeContext *typeContext, const USRBasedTypeContext *usrTypeContext) const;
```

Comment:
`git-clang-format`?

Reply:
Heh, thanks. I meant to do that before posting but forgot :)